website/src/views/JoinClass/JoinClass.test.jsx

```diff
@@ -2,15 +2,34 @@ import React from 'react';
 import renderer from 'react-test-renderer';
 import {
   configure,
+  shallow,
 } from 'enzyme';
 import Adapter from 'enzyme-adapter-react-16';
 import { JoinClass } from '..';
 
 configure({ adapter: new Adapter() });
 
 
+/* Snapshot */
 describe('JoinClass', () => {
   it('Render JoinClass without any eror', () => {
     expect(renderer.create(<JoinClass />).toJSON()).toMatchSnapshot();
   });
 });
+
+/* Enzyme */
+const wrapper = shallow(<JoinClass />);
+describe('JoinClass shallow', () => {
+  it('Join-class-label name should not be change', () => {
+    expect(wrapper.find('.join-class-label').every('.join-class-label')).toBe(true);
```

Comment:
I don't understand what you are trying to do here can you please clarify the logic seems confusing

Reply:
Also the comments seem excessive you don't need to clarify simple things like enzyme

Reply:
There are several join-class-label class being used, I want to make sure no one will change the name of the class when someone trying to reuse the class.

You mean I can remove the /* Enzyme */ and /* Snapshots */ ?

Reply:
I don't think you have to remove /*enzyme*/ and /*snapshots*/. We use them to separate different testing section

Reply:
I dont understand how the class name will be changed If we define it is a such. Besides this test will always pass since you explicitly search for  `.join-class-label` when .every is run it will always be true since you only returned the matching `.join-class-label` from your find function.

Reply:
@khelif96 You're right, I have removed this test case.